corehq/apps/hqcase/management/commands/delete_cases.py

```diff
@@ -16,10 +16,6 @@ class Command(BaseCommand):
                     action='store_true',
                     dest='no_prompt',
                     help='Delete cases without prompting for confirmation'),
-        make_option('--by-last-submitter',
-                    action='store_true',
-                    dest='last_submitter',
-                    help='delete cases last updated by the specified user'),
```

Comment:
Yeah, I doubt it's been used at all.  I dont' even remember what prompted me to make this command or that option.